concent_api/core/tests/utils.py

```diff
@@ -204,6 +205,16 @@ def _test_database_objects(
             self.assertEqual(ReceiveStatus.objects.last().delivered,        receive_delivered_status)
             self.assertEqual(ReceiveStatus.objects.last().message.task_id,  task_id)
 
+    def _test_subtask_state(self, task_id, subtask_id, subtask_state, provider_key, requestor_key, next_deadline = None):
```

Comment:
There is no checking of existence of related `StoredMessages`.

Reply:
Something like `self.assertTrue(StoredMessage.objects.filter(subtask_id = subtask_id).exists())` will be enough? Because there can be different amount of StoredMessage with the same `subtask_id` depending on usecase.

Reply:
No, we need to check exactly what messages are assigned at each state. There are related objects so can be reached by calling ie. `subtask.task_to_compute` etc.